schema/spacewalk/susemanager-schema.spec

```diff
@@ -103,6 +103,9 @@ else
 fi
 %endif
 
+%posttrans
+systemctl try-restart uyuni-check-database.service ||:
```

Comment:
As I don't see this services here, I guess it was added in the past, right?

This looks good, the package will not break as we always return true, so there will be this error, but services will not start.

Reply:
yes, and there are now dependencies which turn tomcat, apache, etc. off if this fail.